label_studio/tasks/models.py

```diff
@@ -577,6 +577,13 @@ def has_permission(self, user):
         user.project = self.task.project  # link for activity log
         return self.task.project.has_permission(user)
 
+    def save(self, *args, **kwargs):
```

Comment:
Maybe this is a wider refactor since we'd have to do this for non-draft labels... but maybe we can make `update_created_labels_drafts` async?